templates/stats/issues.tpl

```diff
@@ -0,0 +1,199 @@
+									@click="setTimelineInterval('month')"
+								>
+									{translate key="stats.monthly"}
+								</pkp-button>
+							</div>
+						</div>
+					</div>
+					<table class="-screenReader" role="region" aria-live="polite">
+						<caption v-if="timelineType === 'files'">{translate key="stats.issue.totalIssueGalleyViews.timelineInterval"}</caption>
+						<caption v-else>{translate key="stats.issue.totalTOCViews.timelineInterval"}</caption>
+						<thead>
+							<tr>
+								<th scope="col">{translate key="common.date"}</th>
+								<th v-if="timelineType === 'files'" scope="col">{translate key="stats.issueGalleyViews"}</th>
+								<th v-else scope="col">{translate key="stats.issueTOCViews"}</th>
+							</tr>
+						</thead>
+						<tbody>
+							<tr	v-for="segment in timeline" :key="segment.date">
+								<th scope="row">{{ segment.label }}</th>
+								<td>{{ segment.value }}</td>
+							</tr>
+						</tbody>
+					</table>
+					<line-chart :chart-data="chartData" aria-hidden="true"></line-chart>
+					<span v-if="isLoadingTimeline" class="pkpStats__loadingCover">
+						<spinner></spinner>
+					</span>
+				</div>
+				<div class="pkpStats__panel" role="region" aria-live="polite">
+					<pkp-header>
+						<h2 id="issueDetailTableLabel">
+							{translate key="stats.issue.details"}
+							<spinner v-if="isLoadingItems"></spinner>
+						</h2>
+						<template slot="actions">
+							<div class="pkpStats__itemsOfTotal">
+								{{
+									replaceLocaleParams(itemsOfTotalLabel, {
+										count: items.length,
+										total: itemsMax
+									})
+								}}
+								<a
+									v-if="items.length < itemsMax"
+									href="#issueDetailTablePagination"
+									class="-screenReader"
+								>
+									{translate key="common.pagination.label"}
+								</a>
+							</div>
+							<pkp-button
+								aria-describedby="report-type-issues report-type-issues-description"
+								@click="downloadReport"
+							>
+								{translate key="common.downloadReport"}
+							</pkp-button>
+						</template>
+					</pkp-header>
+					<pkp-table
+						labelled-by="issueDetailTableLabel"
+						:class="tableClasses"
+						:columns="tableColumns"
+						:rows="items"
+						:order-by="orderBy"
+						:order-direction="orderDirection"
+						@order-by="setOrderBy"
+					>
+						<search
+							slot="thead-title"
+							class="pkpStats__titleSearch"
+							:search-phrase="searchPhrase"
+							search-label="{translate key="stats.issue.searchIssueDescription"}"
+							@search-phrase-changed="setSearchPhrase"
+						></search>
+						<template slot-scope="{ row, rowIndex }">
+							<table-cell
+								v-for="(column, columnIndex) in tableColumns"
+								:key="column.name"
+								:column="column"
+								:row="row"
+								:tabindex="!rowIndex && !columnIndex ? 0 : -1"
+							>
+								<template v-if="column.name === 'title'">
+									<a
+										:href="row.issue._href"
```

Comment:
`row.issue._href` will lead to the API endpoint for this issue. I think what you want is the URL to the published issue. For that, you will need to modify `APP\issues\maps\Schema::mapToStats()`.